src/Hamiltonians/GutzwillerSampling.jl

```diff
@@ -110,3 +120,59 @@ function Base.getindex(h::GutzwillerOffdiagonals{F,T,A,G}, i)::Tuple{F,T} where
 end
 
 Base.size(h::GutzwillerOffdiagonals) = size(h.offdiagonals)
+
+"""
+    similarity_transform(k::GutzwillerSampling, op::AbstractHamiltonian)
+    similarity_transform(k::GutzwillerSampling)
+
+For a Gutzwiller similarity transformation `k = f H f^{-1}` define the transformed
+operator `f op f` for arbitrary `op`, and special case `f^2`, in order to calculate observables. 
```

Comment:
```suggestion
For a Gutzwiller similarity transformation `k = f H f^{-1}` define the transformed
operator `f^{-1} op f^{-1}` for arbitrary `op`, and special case `f^{-2}`, in order to calculate observables. 
```
Don't you need the inverses of the guiding function here?

Reply:
Oh yes you are correct. I have made the docs consistent now.